galaxy_importer/loaders/collection.py

```diff
@@ -99,6 +105,58 @@ def load(self):
             requires_ansible=self.requires_ansible,
         )
 
+    def _lint_collection(self):
+        """Log ansible-lint output.
+
+        ansible-lint stdout are linter violations, they are logged as warnings and errors,
+        depending on the rule level.
+
+        ansible-lint stderr includes info about vars, file discovery,
+        summary of linter violations, config suggestions, and raised errors.
+        Only raised errors are logged, they are logged as errors.
```

Comment:
Originally there were a lot of extra lines in stderr that did not make sense to list in the galaxy-importer log, so we only logged those that started with `constants.ANSIBLE_LINT_ERROR_PREFIXES` - is this still the case?